modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -11,7 +11,62 @@ Before you install an {product-title} cluster, review the following Nutanix AOS
 [id="installation-nutanix-installer-infra-reqs-account_{context}"]
```

Comment:
Expand parent module's use of AOS? AOS =  Acropolis Operating System ?

Reply:
Not necessarily @dfitzmau. In this case, I believe AOS to be an apronym. AOS is well understood by our target audience (Nutanix admins)

Reply:
If you think this is common knowledge among the target audience. NP.